Makefile

```diff
@@ -8,4 +8,7 @@ release:
 	mv puma-dev_darwin_amd64 puma-dev
 	zip puma-dev-$$RELEASE-darwin-amd64.zip puma-dev
 
+test:
```

Comment:
This shouldn't be necessary, but added it in while I try to work out why it doesn't work